src/snowflake/snowpark/_internal/analyzer/analyzer_utils.py

```diff
@@ -1655,3 +1665,246 @@ def cte_statement(queries: List[str], table_names: List[str]) -> str:
         for query, table_name in zip(queries, table_names)
     )
     return f"{WITH}{result}"
+
+
+def write_arrow(
```

Comment:
can you create a jira and add todo here to port this implementation to python connector later?